docs/sources/installation/sizing/index.md

```diff
@@ -15,7 +15,8 @@ keywords: []
 <!-- vale Grafana.Quotes = YES -->
 
 This tool helps to generate a Helm Charts `values.yaml` file based on specified
- expected ingestion, retention rate and node type.
+ expected ingestion, retention rate and node type. It will always configure a
+ [scalable](../../fundamentals/architecture/deployment-modes/#simple-scalable-deployment-mode) deployment. The storage needs to be configured after generation.
```

Comment:
The storage needs to be configured after generation. ->
You need to configure the storage after generating the initial file.

Also, the comment about storage is unclear if the tool estimates storage?  Or is that an additional manual step that you have to perform in the generated file.  In which case something more like:
"After generating the initial file, you must add storage configuration." in which case, we might need examples or links to where this is already documented.

Reply:
Should we also give an instruction here?

"Enter your estimate values.  Hover over each field for additional information.   Then click Generate and download values file."